site/en/docs/extensions/mv3/external_extensions/index.md

```diff
@@ -63,7 +88,19 @@ id", width="800", height="165" %}
 
 <!-- Add screenshot -->
 
-### Installing from Local CRX file {: #prereq-crx }
+### Installing a CRX file in Chrome {: #chrome-crx }
```

Comment:
I don't think these instructions should go in the "prerequisites " section for Alternative installation flows for mass distribution. This article covers ways to install an extension for many users.

The "Installing a CRX file in Chrome" are instructions to install one copy locally.

These instructions are also located in the [view warnings](https://developer.chrome.com/docs/extensions/mv3/permission_warnings/#view_warnings) section.

I can provide more context since I worked on the content of this article.

Reply:
Do you have an idea where to put them?